specification/amqp_0.9.1/src/main/scripts/org/kaazing/specification/amqp/tcp/open/tcp.accept.rpt

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Do you really need to keep these files separate? I did suggest he might just include the accept or connect in the other scripts